src/main/java/bio/terra/janitor/service/cleanup/flight/GoogleProjectCleanupStep.java

```diff
@@ -24,9 +30,18 @@ public GoogleProjectCleanupStep(ClientConfig clientConfig, JanitorDao janitorDao
   @Override
   protected StepResult cleanUp(CloudResourceUid resourceUid) {
     CloudResourceManagerCow resourceManagerCow;
+
     try {
       resourceManagerCow =
-          CloudResourceManagerCow.create(clientConfig, GoogleCredentials.getApplicationDefault());
+          new CloudResourceManagerCow(
+              clientConfig,
+              new CloudResourceManager.Builder(
```

Comment:
le'ts also set the application name, like CRL does for the factory function.
https://github.com/DataBiosphere/terra-cloud-resource-lib/blob/348bb8896893b8374e37ee1935d6eff07b82a0bb/google-cloudresourcemanager/src/main/java/bio/terra/cloudres/google/cloudresourcemanager/CloudResourceManagerCow.java#L50